### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by SAGARPUJAR**
Hey Guys Please Check and let me know if anything to be added or improved

### CODE REVIEW

File: Alert.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  border-color: #e2e3e5;
+}
+
+.alert-success {
+  color: #059669;
+  background-color: #a7f3d0;
+  border-color: #a7f3d0;
+}
+
+.alert-Warning {
```
Comment (humzahasan) [inline Alert.css:52]: Should be all small letters

File: Alert.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <link rel="stylesheet" href="Alert.css" />
+  </head>
+  <body>
+    <main class="header">
+      <h3>Alert Mesages</h3>
+    </main>
+    <div class="container">
+      <div class="alert alert-danger">
+        <span
+          class="closebtn"
```
Comment (humzahasan) [inline Alert.html:17]: Rather than closeBtn something like alert-close or similar will bring consistency

**Review by humzahasan — COMMENTED** [review]
Code looks good. Refactor the above two comments. 

Great selection of class name which explains it purpose. ✅

File: Alert.html (hunk 2 above)
Comment (SAGARPUJAR) [reply Alert.html:17]: Sure thanks Humza Hasan I'll update

**Pull request closed without merging**